packages/frontend/src/app/App.js

```diff
@@ -618,11 +618,7 @@ class Routing extends Component {
                             <PrivateRoute
                                 exact
                                 path="/sign"
-                                render={() => (
-                                    <SignWrapper
-                                        urlQuery={parse(this.props.router.location.hash)}
-                                    />
-                                )}
+                                component={SignWrapper}
```

Comment:
SignWrapper identifies if it's a private shard using the [account slice selector](https://github.com/mynearwallet/near-wallet/pull/102/files#diff-919158a3c721c3eef8638b3be2d3eeb8e1cd47e9f5ed3663e15a2700ffc61a1f)